DeAceroDLabs/marketplace-frontend
# Feature/dmp 492

## What is done

[[DMP-492] Navigation arrows on categories](https://deacero.atlassian.net/browse/DMP-492)

## How is done

- Updated back.
- Changed `nickname` to `given_name`, nickname is the first part of the email while given name is the real name that user gives at sign up.

## How to test

1. Update back: git pull origin master 
```
2. docker compose down
3. docker ps -a
4. docker rm mkt-place-back-postgres-1
5. docker rm mkt-place-back-postgres-1
6. docker images -a
7. docker image rm image id
8. docker-compose up
```
9. Run the project via `npm start`
10. signup
11. login
12. You sould see Hola, *your name*.
